### PR TITLE
Filter index

### PR DESCRIPTION
**Pull request by Robstaa**
**2 Bug fixes:** Index page is now not too long when filter window is not displayed (quick fix) and you cannot send a friendrequest, when you already send one to the same person.

The filter works now on the index page, not the graphic one, but the backend.
**TRY IT OUT on the bottom of the index page** 

--> Right now geofilter and interest filter DO NOT work together, that has still to be fixed

### CODE REVIEW

File: app/assets/stylesheets/components/_navbar.scss [hunk 1]
```diff
@@ -86,3 +86,7 @@
   border-left: 1px solid #E6E6E6;
   border-top: 1px solid #E6E6E6;
 }
+
+.row {
```
Comment (Kalle444) [inline app/assets/stylesheets/components/_navbar.scss]: If you change the bootstrap class row - will this not change on other pages where the bootstrap row has been used?? could you name it something more specific?

File: app/controllers/profiles_controller.rb [hunk 2]
```diff
@@ -4,10 +4,18 @@ class ProfilesController < ApplicationController
   def index
     @location = params[:location]
     @radius = 20
+    @interests = Interest.all
+    @selected_interests = params[:selected_interests]
     @profiles = return_profiles_based_on_location(@location, @radius)
+    if !@selected_interests.nil?
```
Comment (Kalle444) [inline app/controllers/profiles_controller.rb]: Is there a reason for the duble negative: if !@selected_interests.nil?

could you consider having the following, to make it even easier to read? - or am I missing something?

if @selected_interests.nil?
      @profiles_by_interest = @profiles
    else
      @profiles_by_interest = return_profiles_based_on_interests(@profiles)
    end

**Review by Kalle444 — CHANGES_REQUESTED** [review]

File: app/controllers/profiles_controller.rb (hunk 2 above)
Comment (Kalle444) [reply app/controllers/profiles_controller.rb]: you obviously does not have to change if you do not agree - as it works as it is :D

File: app/assets/stylesheets/components/_navbar.scss (hunk 1 above)
Comment (Kalle444) [reply app/assets/stylesheets/components/_navbar.scss]: not 100% sure why this change was made in the navbar? But if you want to change the row - could you then maybe make it nested 

.parent_div {
   .row {
     margin: 0;
    }
}

**Review by Kalle444 — CHANGES_REQUESTED** [review]

File: app/controllers/profiles_controller.rb (hunk 2 above)
Comment (Robstaa) [reply app/controllers/profiles_controller.rb]: I was just too lazy to switch it out so i put an exclamation mark in front, gonna fix that now :p

**Review by Robstaa — COMMENTED** [review]
Deleted the .row css class and switched out double negative function in profiles controller

**Review by Kalle444 — APPROVED** [review]

**Pull request merged**